ballerina-tests/tests/http2_mutual_ssl_test.bal

```diff
@@ -97,8 +97,63 @@ http:ClientConfiguration http2MutualSslClientConf = {
 };
 
 @test:Config {}
-public function testHttp2MutualSsl() {
-    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf);
+public function testHttp2MutualSsl1() {
+    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf1);
+    http:Response|error resp = httpClient->get("/http2Service/");
+    if (resp is http:Response) {
+        assertTextPayload(resp.getTextPayload(), "Passed");
+    } else {
+        test:assertFail(msg = "Found unexpected output: " +  resp.message());
+    }
+}
```

Comment:
can be simplified to 

```ballerina
public function testHttp2MutualSsl1() returns error? {
    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf1);
    http:Response resp = check httpClient->get("/http2Service/");
    assertTextPayload(resp.getTextPayload(), "Passed");
}
```

Reply:
Yes. Will do a simplification across the complete module in a separate PR.

Reply:
The issue is create to track this. https://github.com/ballerina-platform/ballerina-standard-library/issues/1718